test/functional/apps/discover/group1/_sidebar.ts

```diff
@@ -657,6 +657,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         await testSubjects.missingOrFail('discoverNoResultsError');
       });
 
+      // run this test only
```

Comment:
this failure is high likely caused by the unmapped fields.

Adding some debugs here showed

`Error: expected '0 available fields. 46 unmapped fields. 7 empty fields. 3 meta fields.' to equal '0 available fields. 7 empty fields. 3 meta fields.'`

it's the 46 unmapped fields caused it to failed. And those are high likely mapped fields of the previous data view. when those are cleaned up or the unmapped fields are overwritten when the new incoming data view fields are available, it should work